packages/mgt-chat/src/utils/mentions.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { PersonCardInteraction } from '@microsoft/mgt-components';
+import { MgtTemplateProps, Person } from '@microsoft/mgt-react';
+import { ChatMessageMention, User } from '@microsoft/microsoft-graph-types';
+import { GraphChatClient } from 'src/statefulClient/StatefulGraphChatClient';
+import { Mention } from '@azure/communication-react';
+
+export const renderMGTMention = (chatState: GraphChatClient) => {
+  return (mention: Mention, defaultRenderer: (mention: Mention) => JSX.Element): JSX.Element => {
+    let render: JSX.Element = defaultRenderer(mention);
+
+    const mentions = chatState?.mentions ?? [];
+    const flatMentions = mentions?.flat();
```

Comment:
mentions here will never be nullish thanks to the nullish coalescing on the prior line. 
This also means that flatMentions will never be nullish.